kotlin/android/app/build.gradle

```diff
@@ -114,14 +114,11 @@ dependencies {
 
     // Moshi
     implementation "com.squareup.moshi:moshi-kotlin:1.12.0"
+    implementation 'com.squareup.moshi:moshi:1.12.0'
```

Comment:
Do we need both gson and moshi or can we consolidate?

Reply:
no, I will remove the gson and retrofit integration in a new PR.